components/org.wso2.carbon.data.provider/src/main/java/org/wso2/carbon/data/provider/internal/DataProviderAPI.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we update the license year to 2018 here as well..

Reply:
See [above comment](https://github.com/wso2/carbon-analytics/pull/1226/files#r184599792).